### PR TITLE
About Window

### LINKED ISSUE #31: `About`  Window
⚠ This Issue should be done entirly on [about_window](https://github.com/Stake-And-Rope/market-app/tree/about_window) branch

We will have to create a separate `About` window which will display:

- Some basic information about the application
- Application version
- Creators
- GitHub Link
- ....

Start with the basic shell. Create new file named `about.py`. Inside the file create the class `About` and init it's properities. The about window will consist of one main layout and one child vertical layout, where all the objects will be created. You can use [login](https://github.com/Stake-And-Rope/market-app/blob/development/pyqt5-gui/login.py) as reference. 

🎗 All the files, related to the GUI should be created in the [pyqt5-gui](https://github.com/Stake-And-Rope/market-app/tree/development/pyqt5-gui) folder

### PR DESCRIPTION
**Pull request by techfrapi**
Related to #32

### CODE REVIEW

File: pyqt5-gui/about.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+)
+from PyQt5.QtGui import *
+from PyQt5.QtCore import *
+
+
+class About(QWidget):
+    def __init__(self):
+        super().__init__()
+        self.setWindowTitle("About")
+        self.setGeometry(650, 300, 500, 200)
```
Comment (karastoyanov) [inline pyqt5-gui/about.py]: Change the window size to 500/300pxs

**Review by karastoyanov — APPROVED** [review]

File: pyqt5-gui/about.py (hunk 1 above)
Comment (karastoyanov) [reply pyqt5-gui/about.py]: Resolved with [10c00e7](https://github.com/Stake-And-Rope/market-app/pull/33/commits/10c00e7b104d37b4baf95ac5e1c0f8f312cb755f)

**Comment by techfrapi** [comment]
Last commit https://github.com/Stake-And-Rope/market-app/commit/9f872eeb627d7feffe8952c59434c9011e6780aa
Added:
- Story About our company
- Contact us information
- Contributors - with names and link to Github

File: pyqt5-gui/about.py [hunk 2]
```diff
@@ -15,8 +15,67 @@
 class About(QWidget):
     def __init__(self):
         super().__init__()
-        self.setWindowTitle("About")
-        self.setGeometry(650, 300, 500, 300)
+        self.setWindowTitle("About Us")
+        self.setGeometry(650, 300, 600, 400)
+
```
Comment (karastoyanov) [inline pyqt5-gui/about.py:20]: hgf

File: pyqt5-gui/about.py [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+class About(QWidget):
+    def __init__(self):
+        super().__init__()
+        self.setWindowTitle("About Us")
+        self.setGeometry(650, 300, 600, 400)
+
+        main_layout = QVBoxLayout()
+
+        """Init the our story layout"""
```
Comment (karastoyanov) [inline pyqt5-gui/about.py]: Change the comments lines to upper letters only

File: pyqt5-gui/about.py [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+from PyQt5.QtGui import *
+from PyQt5.QtCore import *
+
+
+class About(QWidget):
+    def __init__(self):
+        super().__init__()
+        self.setWindowTitle("About Us")
+        self.setGeometry(650, 300, 600, 400)
+
```
Comment (karastoyanov) [inline pyqt5-gui/about.py:35]: use the `.setWindowIcon()` method to set the window icon. You can use the file `market.png` in img folder

Comment (karastoyanov) [inline pyqt5-gui/about.py:35]: Import custom font using the following block of code

```python
        font = QFontDatabase.addApplicationFont(r'../fonts/jetbrains-mono.regular.ttf')
        if font >= 0:
            fonts = QFontDatabase.applicationFontFamilies(font)
        else:
            print("Error loading fonts!")
```

Then, you can set the custom font to any object below, e.g.

```python
        contact_us = QLabel()
        contact_us.setText("<b><font size='4'>Contact Us</font></b><br>"
                           "123-456-7890<br>"
                           "stake_robe@example.com")
        contact_us.setFont(QFont(fonts[0], 12)) # where 12 is the font size
```

**Review by karastoyanov — CHANGES_REQUESTED** [review]
Check the following code reviews

File: pyqt5-gui/about.py [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+        main_layout.addLayout(contact_us_layout)
+        main_layout.addLayout(contributors_layout)
+
+        self.setLayout(main_layout)
+
+
+app = QApplication(sys.argv)
+about_window = About()
+about_window.show()
+app.exec_()
```
Comment (karastoyanov) [inline pyqt5-gui/about.py:129]: test code review

**Review by karastoyanov — CHANGES_REQUESTED** [review]
take action

**Review by karastoyanov — APPROVED** [review]

**Pull request merged**